README.md

```diff
@@ -117,11 +117,20 @@ Makes sure files end in a newline and only a newline.
 Sort the lines in specified files (defaults to alphabetical).
 You must provide the target [`files`](https://pre-commit.com/#config-files) as input.
 Note that this hook WILL remove blank lines and does NOT respect any comments.
+All newlines will be converted to line feeds (`\n`).
 
 The following arguments are available:
 - `--ignore-case` - fold lower case to upper case characters.
 - `--unique` - ensure each line is unique.
 
+When using this hook in a git repository on Windows it is recommended to set
+[`core.autocrlf`](https://www.git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf)
+to `false` or set the end-of-line to line feed via
+[`gitattributes`](https://www.git-scm.com/docs/gitattributes#_eol) for the
+targeted files. Without these changes the hook will fail, but produce no
+apparent change in `git diff`. See
+[#958](https://github.com/pre-commit/pre-commit-hooks/issues/958) for details.
+
```

Comment:
I would just leave the first part about `\n` -- we don't need a whole tutorial about newlines here

Reply:
Okay, I reverted this part.